lib/deck.rb

```diff
@@ -0,0 +1,56 @@
+require './lib/card'
+require 'pry'
+
+class Deck
+
+    attr_reader :cards
+
+    def initialize(cards)
+
+        @cards = []
+        @cards = cards
+    
+    end
+
+    def rank_of_card_at(index)
+        @cards[index].rank
+    end
+
+    def high_ranking_cards
+
+        high_rank_cards = []
+
+        @cards.each do |card|
+            if card.rank >= 11
+                high_rank_cards << card
+            end
+        end
+
+        high_rank_cards
+
```

Comment:
This works great! When you're ready for a refactor, see if you can solve this using the `select` enumerable method instead.

Reply:
This comment is somewhat confusing for me: do you mean for me to revisit this to try to refactor later on? Or is this something you would like me to do for the assessment?

I'm going to make an educated guess that it's the prior, and that it's for practice, correct?

Reply:
Yes just for practice if you feel like coming back to it!